product_dimension/models/product.py

```diff
@@ -24,11 +23,7 @@ def onchange_calculate_volume(self):
 
     def convert_to_meters(self, measure, dimensional_uom):
         uom_meters = self.env['product.uom'].search([('name', '=', 'm')])
```

Comment:
@gurneyalex I created a PR with these two modification. Do what you want with this PR :p